packages/marshal/src/marshal.js

```diff
@@ -445,13 +462,33 @@ function makeReviverIbidTable(cyclePolicy) {
   });
 }
 
-const identityFn = x => x;
+/**
+ * @template Slot
+ * @type {ConvertValToSlot<Slot>}
+ */
+const defaultValToSlotFn = x => x;
+/**
+ * @template Slot
+ * @type {ConvertSlotToVal<Slot>}
+ */
 const defaultSlotToValFn = (x, _) => x;
 
+/**
+ * @template Slot
+ * @type {MakeMarshal<Slot>}
+ */
 export function makeMarshal(
-  convertValToSlot = identityFn,
+  convertValToSlot = defaultValToSlotFn,
   convertSlotToVal = defaultSlotToValFn,
 ) {
+  /**
+   * @template Slot
+   * @param {Passable} val
+   * @param {Slot[]} slots
```

Comment:
I used to like `Slot[]` better, but found that the JSDoc linter prefers `Array<Slot>`. YMMV.

Reply:
In what way does JSDoc prefer the other?

Other things being equal, I still prefer `Slot[]`